src/AppBundle/Resources/todo/app/templates/components/todo-item.hbs

```diff
@@ -0,0 +1,14 @@
+<li class="todo list-group-item">
+    <span {{action 'toggle' todo}} class="todo-check" style="cursor:pointer;">
```

Comment:
This action doesn't exist anymore. Should be in `app/components/todo-item.js` or added back to `app/routes/todos.js`.